Source/UIKit/UITextField.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  UITextField.swift
+//  Rex
+//
+//  Created by Rui Peres on 17/01/2016.
+//  Copyright © 2016 Neil Pankey. All rights reserved.
+//
+
+import Foundation
+import ReactiveCocoa
+import UIKit
+
+extension UITextField {
+    
+    /// Sends the field's string value whenever it changes.
+    public var rex_textSignal: SignalProducer<String, NoError> {
+        return NSNotificationCenter.defaultCenter()
+            .rac_notifications(UITextFieldTextDidChangeNotification, object: self)
```

Comment:
This looks good though, was worried about the notif center retaining the passed in object which it doesn't seem to be.